### ISSUE TITLE
TypeError: Cannot read property 'maxSize' of undefined

### ISSUE THREAD

**Issue by benjamin-dk** [cfdp/opeka#45]
commit 467e5a8389abffbdd861fe653dac9e70a7c6039f

The chat app crashed today with this error:

2015-10-21T14:23:14.039Z - ^[[31mwarning^[[39m: Last admin user disconnected - shutting down room. Counselor id:  00e88445-bd6c-42ee-be75-1ae35b849a22

/home/chat/opeka/nodejs/lib/opeka.js:993
    else if (autoPause === true && room.maxSize === 2 && !room.paused && (acti
                                       ^
TypeError: Cannot read property 'maxSize' of undefined
    at Server.self.removeUserFromRoom (/home/chat/opeka/nodejs/lib/opeka.js:993:40)
    at /home/chat/opeka/nodejs/lib/opeka.js:936:14
    at Array.forEach (native)
    at /home/chat/opeka/nodejs/lib/opeka.js:920:37
    at process._tickCallback (node.js:415:13)

**Comment by benjamin-dk** [comment]
Fixed in 15c1552cfe8981d69

**Comment by benjamin-dk** [comment]
This does not seem to be fixed:

```
2018-01-15T19:50:52.427Z - warning: Last admin user disconnected - shutting down room. Counselor id:  0f39daf0-44f1-4e42-a7d0-39bf4ded0474
stream.js:74
      throw er; // Unhandled stream error in pipe.
      ^

TypeError: Cannot read property 'maxSize' of undefined
    at Client.<anonymous> (/home/<domain>/domains/<domain>-node.curachat.com/opeka/nodejs/lib/opeka.js:1061:37)
    at methods.(anonymous function) (/home/<domain>/domains/<domain>-node.curachat.com/opeka/nodejs/lib/groups.js:184:49)
    at Proto.apply (/home/ubuntu/.nvm/versions/node/v6.6.0/lib/node_modules/dnode/node_modules/dnode-protocol/index.js:123:13)
    at Proto.handle (/home/ubuntu/.nvm/versions/node/v6.6.0/lib/node_modules/dnode/node_modules/dnode-protocol/index.js:99:19)
    at D.dnode.handle (/home/ubuntu/.nvm/versions/node/v6.6.0/lib/node_modules/dnode/lib/dnode.js:140:21)
    at D.dnode.write (/home/ubuntu/.nvm/versions/node/v6.6.0/lib/node_modules/dnode/lib/dnode.js:128:22)
    at SockJSConnection.ondata (stream.js:31:26)
    at emitOne (events.js:96:13)
    at SockJSConnection.emit (events.js:188:7)
    at Session.didMessage (/home/ubuntu/.nvm/versions/node/v6.6.0/lib/node_modules/shoe/node_modules/sockjs/lib/transport.js:220:25)
error: Forever detected script exited with code: 1
error: Script restart attempt #1
2018-01-15T19:51:07.826Z - info: Connecting to database...
```
